packages/sdk/src/lend/providers/morpho/index.ts

```diff
@@ -123,7 +119,10 @@ export class LendProviderMorpho extends LendProvider {
             value: 0n,
           },
         },
-        slippage: options?.slippage || this.defaultSlippage,
+        slippage:
+          options?.slippage ||
+          this._config.defaultSlippage ||
```

Comment:
nit: you could have `this._config.defaultSlippage` always be defined and default to `DEFAULT_VERBS_CONFIG.lend.defaultSlippage` that way it always has a default value when it isnt defined